HuntBuddy/Windows/MainWindow.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+using Dalamud.Interface;
+using Dalamud.Interface.Windowing;
+using ImGuiNET;
+using HuntBuddy.Utils;
+
+namespace HuntBuddy.Windows;
+
+/// <summary>
+/// Main plugin window.
+/// </summary>
+public class MainWindow : Window
+{
+    public MainWindow() : base(
+        $"{Plugin.Instance.Name}",
+        ImGuiWindowFlags.NoDocking,
+        true)
+    {
+        this.Size = new Vector2(400 * ImGui.GetIO().FontGlobalScale, 500);
+        this.SizeCondition = ImGuiCond.Once;
+
+        if (Plugin.Instance.Configuration.LockWindowPositions)
```

Comment:
This also needs to go in the pre-draw function, same issue as the config window

Reply:
Addressed in 93872f1